websites/S/SVT Play/presence.ts

```diff
@@ -24,13 +24,13 @@ presence.on("UpdateData", async () => {
 			presenceData.details = "Navigerar program kategorier";
 		} else if (document.location.pathname.includes("/kanaler/")) {
 			const video = document.querySelector<HTMLVideoElement>(
-				"#play_main-content > article > div.play_channels-video--show > div > div > div.play_video-player.lp_video.play_channels__active-video > div > video"
+				'[data-rt="video-player-channels"]'
 			);
 			title = document.querySelector(
-				"#play_main-content > article > div.play_channels-video--show > div > div > div.play_channels__active-video-info > h2"
+				"#play_main-content > div > div.sc-b35938a7-1.bTCBgw > div > div > div.sc-2c367a6f-0.dxLhHV > h2"
```

Comment:
See if there is a more general selector that can be used here for presence stability

Reply:
I'm not sure if this would be general enough, but at the very least, it would be better than going after specific classes—that's for sure.

```suggestion
				"#play_main-content > div > div > div > div > div > h2"
```